setup.py

```diff
@@ -107,6 +107,7 @@ def get_required_packages(requirement_files: list[str]) -> list[str]:
     packages=find_packages(where="src", include=["anomalib", "anomalib.*"]),
     install_requires=INSTALL_REQUIRES,
     extras_require=EXTRAS_REQUIRE,
+    include_package_data=True,
     package_data={"": ["config.yaml"]},
```

Comment:
From what I tested
```package_data={"": ["config.yaml"], "anomalib": ["py.typed"]},```
should be sufficient

Reply:
Yes, I've tested it as well. It worked. But if we are going to use `MANIFEST.in`, we need to use the include_package_data=True.
If we are going to use `package_data={"": ["config.yaml"], "anomalib": ["py.typed"]},`, then we need to remove `MANIFEST.in`, because they have duplicated functionality, which is to include the same file.